apps/costcontrol/js/config/config_manager.js

```diff
@@ -364,7 +364,8 @@ var ConfigManager = (function() {
     // XXX: Once loaded via requestConfiguration() / requestAll() it is ensured
     // it wont change so you can use this to access OEM confguration in a
     // synchronous fashion.
-    get configuration() { return configuration; }
+    get configuration() { return configuration; },
+    supportCustomizeMode: false
```

Comment:
File a bug saying `Remove supportCustomizeMode flag when related functionallity is landed and well tested` and add a referrence to that new file here, please.